_ml-commons-plugin/cluster-settings.md

```diff
@@ -47,49 +45,44 @@ plugins.ml_commons.task_dispatch_policy: round_robin
 
 ## Set sync up job intervals 
```

Comment:
Global: Do we have to use "sync-up job" (note the hyphen), or can we call it something else? "Sync-up job" is awkward/clunky.

Reply:
Does "sync job" sound better?

Reply:
Yes